topics/persistence.md

```diff
@@ -203,6 +222,25 @@ corruption happened to be in the initial part of the file.
 Log rewriting uses the same copy-on-write trick already in use for
 snapshotting.  This is how it works:
 
+**Redis >= 7.0**
+
+* Redis [forks](http://linux.die.net/man/2/fork), so now we have a child
+and a parent process.
+
+* The child starts writing the new AOF in a temporary file.
+
+* The parent opens a new increments AOF file to continue writing updates.
+  If the rewriting fails, the old base and increments files (if we have) plus this newly opened increments file represents the complete updated dataset,
+  so we are safe.
+  
+* When the child is done rewriting the file, the parent gets a signal,
+and use newly opened increment file and child generated base file to build an temp manifest,
+and persist it.
+
+* Profit! Now Redis do an atomic exchange to the manifest pointer in the memory, so that the result of this AOF rewrite is visible to Redis.
```

Comment:
```suggestion
* Profit! Now Redis does an atomic exchange of the manifest files so that the result of this AOF rewrite takes effect. Redis also cleans up the old base file, manifest file and any unused increment files.
```